fetools-app/src/pages/ColorPicker.jsx

```diff
@@ -28,12 +28,13 @@ export default function ColorPicker() {
   const [colorData, setColorData] = useState(
     createColorObj(searchParams.get('color')) || createColorObj()
   );
-
-  const toastState = useToastState();
-
+  
   const [bookmarkLength, setBookmarkLength] = useState(checkForLocalStorage('colors').length)
   const [isExpanded, toggleIsExpanded] = useExpander();
+  
+  const toastState = useToastState();
 
+    console.log(compareItems(getColorString(colorData.color, 'hex'), 'colors', 'color'))
```

Comment:
This is a solid step! We can simplify further by setting state isBookmarked based on the comparison. Then the isBookmarked state can be passed down to the bookmark button and used to determine button presentation and action.